tmt/steps/provision/mrack.py

```diff
@@ -596,6 +596,23 @@ def _transform_system_model_name(
                         children=[MrackHWBinOp('model', beaker_operator, actual_value)])
 
 
+def _transform_system_vendor_name(
+        constraint: tmt.hardware.NumberConstraint,
```

Comment:
Shouldn't it be `TextConstraint` instead of  `NumberConstraint`?
```suggestion
        constraint: tmt.hardware.TextConstraint,
```

Reply:
yeah, sorry for failing to notice that by myself,and update:)